Makefile

```diff
@@ -21,9 +23,14 @@ _bootstrap-node:
 	yarn config set cache-folder ${YARN_CACHE_DIR}
 	yarn install
 
+.PHONY: _bootstrap-python
+_bootstrap-python:
+	PIPENV sync
```

Comment:
Hmm, does this work? Would need to be `${PIPENV}` to evaluate the variable right?